cfme/infrastructure/config_management.py

```diff
@@ -154,8 +156,22 @@ def is_displayed(self):
                 self.context['object'].type == 'Ansible Tower'):
             title_text = 'All Ansible Tower Providers'
         else:
-            title_text = 'All Configuration Manager Providers'
-        return self.in_config and self.entities.title.text == title_text
+            title_text = 'All Configuration Management Providers'
+        return (
+            self.in_config and
+            self.entities.title.text == title_text
+        )
+
+
+class ConfigSystemAllView(ConfigManagementAllView):
+    """The config system view has a different title"""
+
+    @property
+    def is_displayed(self):
+        return (
+            self.in_config and
+            self.entities.title.text == 'All Configured Systems'
```

Comment:
Q: if self.context['object'].type == 'Ansible Tower', then title will be 'All Ansible Tower Configured Systems'?